toml/longrun_1M.toml

```diff
@@ -0,0 +1,15 @@
+[zd_rayleigh]
+value = 35000.0
+type = "float"
+
+[alpha_rayleigh_uh]
+value = 0.0
+type = "float"
+
+#[rain_autoconversion_timescale]
```

Comment:
Is this commented block intentional? It seems the default values are 1000s, 100s, for the `rain`, `snow` vars respectively.

Reply:
Yes, but I can delete it for now. I don't know yet what would be best/most stable for the Atmos runs. I just wanted to remember what are the parameter names to test them later.